doc/book/getting-started-with-zend-studio/the-application.md

```diff
@@ -549,7 +548,7 @@ return [
         'driver' => 'Pdo',
         'dsn' => 'mysql:dbname=mytasklist;hostname=localhost',
         'driver_options' => [
-            PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES \'UTF8\''
+            PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES \'UTF8\'',
```

Comment:
Please remove this. [It's no longer needed, use `charset` in `dsn` instead.](http://php.net/manual/en/ref.pdo-mysql.connection.php#refsect1-ref.pdo-mysql.connection-changelog)